### ISSUE TITLE
Double click mouse event clashes in echart panel 

### ISSUE THREAD

**Issue by Annexhc** [mzmine/mzmine2#654]
Hello,
the EChartPanel causes for some annoying clashes using double click mouse events. For example the double click gesture is assigned twice in the TICVisualizer, zoom history back, and open spectra. I just wanted to start a small discussion how we should change the gestures. I really like the zoom history, but I think opening e.g. a spectra with double click is as important or even more important. We could go for triple click for zoom history back. Any other ideas?

**Comment by robinschmid** [comment]
Hey Ansgar,

The chart gestures can be changed for each single chart. 
I have no plan how to change this problem without having a different behaviour compared to the other charts.
No one will use a tripple click

**Comment by tomas-pluskal** [comment]
I agree that tripple click is not very intuitive. Maybe we could add the zoom history controls to the popup menu?
